src/Database/Database.php

```diff
@@ -1504,6 +1526,7 @@ public function select($table, array $fields = [], array $condition = [], array
 	 */
 	public function count($table, array $condition = [], array $params = [])
 	{
+		// @TODO Can we dump this to have ": int" as returned type-hint?
```

Comment:
Yes. The error state is taken care of in line 1550 anyway. A better solution would be to throw an `InvalidArgumentException` which should never trigger since it'd be a developer error.

Reply:
But I guess this process need to be done slowly.